qtism/data/storage/xml/marshalling/OutcomeDeclarationMarshaller.php

```diff
@@ -94,6 +94,10 @@ protected function marshall(QtiComponent $component) {
 			$lookupTableMarshaller = $this->getMarshallerFactory()->createMarshaller($component->getLookupTable(), array($component->getBaseType()));
 			$element->appendChild($lookupTableMarshaller->marshall($component->geTLookupTable()));
 		}
+
+		if ($component->getExternalScored() !== null) {
+		    static::setDOMElementAttribute($element, 'externalScored', $component->getExternalScored());
```

Comment:
do u have a constant for that :) ?

Reply:
No. Should I?

Reply:
cf @julien-sebire

Reply:
I think we could stay consistent for this for the moment.
There's a lot of cleaning I'm doing now to merge legacy branch back into develop, and I'd like to keep the changes at the minimum. We can optimize the code once it's merged.